docs/standard/native-interop/best-practices.md

```diff
@@ -126,7 +134,7 @@ public struct UnicodeCharStruct
 }
 ```
 
-`string` contains blittable contents if it isn't contained in another type and it's being passed as an argument that is marked with `[MarshalAs(UnmanagedType.LPWStr)]` or the `[DllImport]` has `CharSet = CharSet.Unicode` set.
+`string` contains blittable contents if it isn't contained in another type and it's being passed as an argument that is marked with `[MarshalAs(UnmanagedType.LPWStr)]` or `[LibraryImport]` has `StringMarshalling = StringMarshalling.Utf16` or the `[DllImport]` has `CharSet = CharSet.Unicode` set.
```

Comment:
Seems this wording simplification means the same thing?

```suggestion
`string` contains blittable contents if it isn't contained in another type and is being passed as an argument that is marked with `[MarshalAs(UnmanagedType.LPWStr)]` or `[LibraryImport]` has `StringMarshalling = StringMarshalling.Utf16` or the `[DllImport]` has `CharSet = CharSet.Unicode` set.
```

Reply:
You might consider a bullet list here too.